### PR TITLE
Lvjavj 133

### PR DESCRIPTION
**Pull request by sergvasylchak**
Tickets and single-ticket pages changed according to PM recomendations

### CODE REVIEW

File: web/src/main/resources/public/src/app/user/ticket/single_ticket/single.ticket.component.ts [hunk 1]
```diff
@@ -397,4 +400,14 @@ export class MessageComponent implements OnInit {
...
+    setDiscussed(dateSet:Date):void {
+        this.ticket.discussed = dateSet;
+        this.editTicket(this.ticket);
+    }
+
+    isDateRight():boolean {
+        return (( this.ticket.discussed > this.ticket.deadline)||(this.ticket.deadline==null))&&( this.ticket.discussed > new Date());
+    }
+
 }
```
Comment (KostyantynPanchenko) [inline web/src/main/resources/public/src/app/user/ticket/single_ticket/single.ticket.component.ts:418]: add new line, plz

File: web/src/main/resources/public/src/app/user/ticket/single_ticket/single.ticket.component.html [hunk 2]
```diff
@@ -78,37 +78,75 @@
...
+                            {{'optionsTicket'|translate}}
+                            <span class="caret"></span>
+                        </button>
+                        <ul class="dropdown-menu" aria-labelledby="dropdownMenu1">
+                            <li><a style="cursor: pointer" (click)="editForm.initUpdatedTicket(ticket);editForm.openEditModal()">
+                                {{'editTicket' | translate}}</a></li>
+                            <li><a style="cursor: pointer" (click)="delForm.openDelModal(ticket)">{{'deleteTicket'
+                                | translate}}</a></li>
+                        </ul>
                     </div>
```
Comment (KostyantynPanchenko) [inline web/src/main/resources/public/src/app/user/ticket/single_ticket/single.ticket.component.html:100]: to much blank lines in markup, plz delete them

File: web/src/main/resources/public/src/app/user/ticket/single_ticket/message.ts [hunk 3]
```diff
@@ -10,6 +10,8 @@ export interface IMessage {
     user:User;
     answers:Message[];
     discussed:Date;
+    deadline:Date;
+    date:Date;
```
Comment (KostyantynPanchenko) [inline web/src/main/resources/public/src/app/user/ticket/single_ticket/message.ts]: delete blank line, the same in line 29

**Review by KostyantynPanchenko — CHANGES_REQUESTED** [review]
Please make some minor changes and resolve conflicts in i18n files

**Review by KostyantynPanchenko — APPROVED** [review]

**Review by LybomirBliharski — APPROVED** [review]

**Pull request merged**